script/releases

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env node
+
+var fs = require('fs')
+var request = require('request')
+
+var options = {
+  url: 'https://api.github.com/repos/atom/electron/releases',
+  json: true,
+  headers: { 'User-Agent': 'Electron' }
+}
+
+request(options, function (error, response, body) {
+  if (error) return console.log(error)
+  if (!error && response.statusCode == 200) {
```

Comment:
I don't think you need to check for `!error` here, since the line before will return early any time `error` is truthy.